handlers/eth1Block.go

```diff
@@ -119,7 +119,7 @@ func Eth1Block(w http.ResponseWriter, r *http.Request) {
 func GetExecutionBlockPageData(number uint64, limit int) (*types.Eth1BlockPageData, error) {
 	block, err := db.BigtableClient.GetBlockFromBlocksTable(number)
 	if diffToHead := int64(services.LatestEth1BlockNumber()) - int64(number); err != nil && diffToHead < 0 && diffToHead >= -5 {
-		block, _, err = rpc.CurrentErigonClient.GetBlock(int64(number))
+		block, _, err = rpc.CurrentErigonClient.GetBlock(int64(number), "parity/geth")
```

Comment:
there are only two trace types available right now (parity type traces and geth type traces). I'd like to move everything away from flags, so that change can be done in this scope of that refactor.